config/sync/views.view.catalog.yml

```diff
@@ -73,8 +73,8 @@ display:
           columns: 3
           automatic_width: true
           alignment: horizontal
-          col_class_default: true
-          col_class_custom: ''
+          col_class_default: false
```

Comment:
Let's enhance the Frontend experience and use the tools provided by the theme (as the Recipe Theme uses Bootstrap, you can use all the tools available in Bootstrap).

Firstly, don't use the `Grid` format in views as it's providing Drupal specific rules which you then need to override, which is an overhead that we don't want. 
Let's use the `Unformatted List` format instead and then you can provide the Bootstrap classes there, either from a template or the view itself (similarly how you do it here).